pkg/utils/errors.go

```diff
@@ -17,7 +17,6 @@
 package utils
 
 const (
-	// TODO - Remove this
-	// ErrInitializingSecretProvider ...
-	ErrInitializingSecretProvider = "error initlializing secret provider"
+	// ErrDecryptionNotSupported ...
+	ErrDecryptionNotSupported = "API key is encrypted, decryption is supported only by sidecar"
```

Comment:
As per configuration API key is encrypted which is not supported by the `sss` library.

Reply:
Updated with new error message.